st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -63,6 +67,35 @@ def init_vars(self, action_parameters):
             self.vars = self._get_rendered_vars(self.actionchain.vars,
                                                 action_parameters=action_parameters)
 
+    def validate(self):
```

Comment:
> In workflow cases we only can really validate at runtime since the workflow could be changed without StackStorm even knowing. I do that myself all the time.

Yeah, that's a good point. I also utilize this "feature" a lot myself as well (I think this was probably more of a by product of an implementation an not an actual feature, but anyway, people like it :)).

So yeah, for now, I will leave it as it is.